helm/worlds-simplest-service-broker/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: World's Simplest Service Broker for Service Catlog
 name: worlds-simplest-service-broker
-version: 1.3.1
+version: 1.3.2
```

Comment:
CI will bump this when it ships a release

Reply:
Ok

Reply:
@drnic  I decrease the version number as requested